src/app/components/shell/header/header.scss

```diff
@@ -53,6 +55,8 @@
             background-color: os-color(red);
             color: text-color(white);
             padding: 0.5rem $normal-margin;
+            outline: thick solid #1f96a8;
```

Comment:
I'm not sure if os-web has a theme constants file but these colors seem good for reuse?

Reply:
We have color constants in the pattern library (a separate package).
These colors are pretty close to each other, and also close to the library color `medium-blue`, so I made a lightened version of medium blue that meets the contrast requirements for both and used that. We might want to add it to the pattern library if the focus color issue crops up elsewhere.